### PR TITLE
DOCS: Reintroducing APT and YUM installation methods for 22.3

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Reintroducing APT and YUM installation methods in Linux.

### CODE REVIEW

File: docs/install_guides/installing-openvino-yum.md [hunk 1]
```diff
@@ -1,10 +1,210 @@
...
+   For example:
+
+   .. code-block:: sh
+
+      sudo yum autoremove openvino-2022.3.0
+
+@endsphinxdirective
+
+
+### Step 3 (Optional): Install OpenCV from YUM
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-yum.md:138]: 1. Remove OpenCV section
2. Add build samples section as it was written in debian document

File: docs/install_guides/installing-openvino-yum.md [hunk 2]
```diff
@@ -1,10 +1,210 @@
...
+   To perform inference on Intel® Neural Compute Stick 2 powered by the Intel® Movidius™ Myriad™ X VPU, follow the steps on :ref:`NCS2 Setup Guide <ncs guide>`.
+   <!--For more details, see the `Get Started page for Intel® Neural Compute Stick 2 <https://software.intel.com/en-us/neural-compute-stick/get-started>`.-->
+
+.. tab:: VPU
+
+   To install and configure your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, see the :ref:`VPU Configuration Guide <vpu guide>`.
+   After configuration is done, you are ready to run the verification scripts with the HDDL Plugin for your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs.
+
+   .. warning::
+      While working with either HDDL or NCS, choose one of them as they cannot run simultaneously on the same machine.
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-yum.md:192]: remove VPU, NSC2, GNA plugin
they are not available in RPM

File: docs/install_guides/installing-model-dev-tools.md [hunk 3]
```diff
@@ -91,65 +91,67 @@ Note that the commands are different for a Python installation and a C++ install
...
-   
+
       pip install openvino-dev[extras]
-      
-  where the `extras` parameter specifies one or more deep learning frameworks via these values: `caffe`, `kaldi`, `mxnet`, `onnx`, `pytorch`, `tensorflow`, `tensorflow2`. Make sure that you install the corresponding frameworks for your models.
-  
-  For example, to install and configure the components for working with TensorFlow 2.x and ONNX, use the following command:
-  
+
+   where the `extras` parameter specifies one or more deep learning frameworks via these values: `caffe`, `kaldi`, `mxnet`, `onnx`, `pytorch`, `tensorflow`, `tensorflow2`. Make sure that you install the corresponding frameworks for your models.
```
Comment (AlexKoff88) [inline docs/install_guides/installing-model-dev-tools.md:100]: Recently, we came up with a different approach to how to use extras for openvino-dev package. The main idea is to provide a smooth transition from DL framework to openvino. Here is how it should work:
- The user trains/downloads a model in the framework representation (99+%) it is about the Python environment.
- Using a framework model means that a specific version of the framework is installed already.
- The user decides to use OpenVINO framework. To do that it installs openvino-dev package into the same python environment where the framework is installed. No extras are used in this case.
- In case there are any issues with framework-openvino compatibility the user can try to install openvino-dev into a separate environment and use extra to get the specific validated version of the framework.

Hope this approach is clear. If so, can you please to revise this section accordingly?

File: docs/install_guides/installing-openvino-yum.md [hunk 4]
```diff
@@ -1,10 +1,210 @@
...
+
+      yum list 'openvino*'
+
+@endsphinxdirective
+
+### Step 2: Install OpenVINO Runtime Using the YUM Package Manager
+
+OpenVINO™ Runtime will be installed in: `/opt/intel/openvino_<VERSION>.<UPDATE>.<PATCH>`
+
+A symlink will be created: `/opt/intel/openvino_<VERSION>`
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-yum.md:64]: packages are installed to the system locations, not in /opt

File: docs/install_guides/installing-model-dev-tools.md (hunk 3 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-model-dev-tools.md:100]: Thanks for the update @AlexKoff88 ! 
Can we revise this section like this:

- If you have installed your deep learning framework in the same environment as OpenVINO Runtime, run `pip install openvino-dev` to install OpenVINO Development Tools.
- In case that you encounter any compatibility issues between OpenVINO and your deep learning framework, you may install OpenVINO Development Tools into a separate environment and use the following command to get specific validated versions of your framework:
    .. code-block:: sh
 
       pip install openvino-dev[extras]

  where [extras]...

Comment (xu-yuan1) [reply docs/install_guides/installing-model-dev-tools.md:100]: And this only affect the Python installation, right? Do we also need to change the recommendations for C++ users?

File: docs/install_guides/installing-model-dev-tools.md [hunk 5]
```diff
@@ -91,65 +91,67 @@ Note that the commands are different for a Python installation and a C++ install
...
-  
-  1. Install the OpenVINO Development Tools mandatory requirements using the following command:
+   When using OpenVINO Development Tools for C++ development, it’s important to install the same version as OpenVINO Runtime. Following the instructions below will ensure that you are installing a version that matches that of OpenVINO Runtime.
+
+   **Recommended: Install Using the Requirements Files**
+
+   After you have installed OpenVINO Runtime from an archive file, you can find a set of requirements files in the <INSTALL_DIR>\tools\ directory. The requirements files will install the matching version of OpenVINO Development Tools and its dependencies.
+
+   1. Install the OpenVINO Development Tools mandatory requirements using the following command:
+      .. code-block:: sh
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:121]: @sgolebiewski-intel Unfortunately the formatting here is still not correct. :(

File: docs/install_guides/installing-model-dev-tools.md [hunk 6]
```diff
@@ -91,65 +91,67 @@ Note that the commands are different for a Python installation and a C++ install
...
-      pip install -r <INSTALL_DIR>\tools\requirements_tensorflow2.txt
-  
-  **Alternative: Install the openvino-dev Package from PyPI**
-  You can also install OpenVINO Development Tools from PyPI using the following command.
-  
    .. important::
-    Make sure to specify the `openvino-dev` version that matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.
-    
+
+      Make sure to specify the `openvino-dev` version that matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-model-dev-tools.md:136]:
```suggestion
      Make sure that the `openvino-dev` version you specified matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.
```

File: docs/install_guides/installing-openvino-apt.md [hunk 7]
```diff
@@ -1,10 +1,228 @@
...
 
-* [Install OpenVINO Runtime from an Archive File](installing-openvino-from-archive-linux.md)
-* [Install OpenVINO from PyPI](installing-openvino-pip.md)
-* [Install OpenVINO with Docker](installing-openvino-docker-linux.md)
-* [Build From Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)
+> **NOTE**: From the 2022.1 release, OpenVINO™ Development Tools can be installed via PyPI only. See [Install OpenVINO Development Tools](#installing-openvino-development-tools) for more information.
 
-The other installation methods are temporarily unavailable.
\ No newline at end of file
+> **IMPORTANT**: By downloading and using this container and the included software, you agree to the terms and conditions of the [software license agreements](https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf). Please review the content inside the `<INSTALL_DIR>/licensing` folder for more details.
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-apt.md:7]: > We don't have `<INSTALL_DIR>/licensing` inside Debian packages now. Each debian package contains `/usr/share/doc/<package-name>/copyright`, but I'm not sure that we need explicitly write about it.

how about we just remove this disclaimer?

File: docs/install_guides/installing-openvino-apt.md [hunk 8]
```diff
@@ -1,10 +1,228 @@
...
+   ```sh
+   apt-cache search openvino
+   ```
+
+
+### Step 2: Install OpenVINO Runtime Using the APT Package Manager
+
+OpenVINO will be installed in: `/opt/intel/openvino_<VERSION>.<UPDATE>.<PATCH>`
+
+A symlink will be created: `/opt/intel/openvino_<VERSION>`
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-apt.md:77]: > OpenVINO will be installed into system paths

Can you help expand on what "system paths" mean?

File: docs/install_guides/installing-openvino-apt.md [hunk 9]
```diff
@@ -1,10 +1,228 @@
...
+.. tab:: VPU
+
+   To install and configure your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, see the :ref:`VPU Configuration Guide <vpu guide>`.
+   After configuration is done, you are ready to run the verification scripts with the HDDL Plugin for your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs.
+
+   .. warning::
+      While working with either HDDL or NCS, choose one of them as they cannot run simultaneously on the same machine.
+
+@endsphinxdirective
+
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-apt.md:205]: We can remove Step 3 (install OpenCV), so Step 4 (Configure Inference on Non-CPU Devices) becomes the new Step 3, then we add a new "Step 4: Build Samples" with those two commands. Is it correct? Should we also add it as an optional step?

**Comment by xu-yuan1** [comment]
@sgolebiewski-intel We are missing two pages to be updated from the list I sent you: the installation overview page and the openvino_docs_get_started_get_started_demos page.

File: docs/install_guides/installing-model-dev-tools.md (hunk 3 above)
Comment (AlexKoff88) [reply docs/install_guides/installing-model-dev-tools.md:100]: I would rephrase the first part as follows:
* You can install OpenVINO Developer Tools into the existing environment with the deep learning framework of your choice. Run the following command to do that:
```python
pip install openvino-dev
```

Just to be on the same page. The transition from the DL framework starts not from OpenVINO Runtime. You should convert the model first using Developer Tools. Please consider this point.
BTW, dev tools usually stand for Developer Tools, not Development. Hope, we can have the same notation.

File: docs/install_guides/installing-openvino-yum.md [hunk 10]
```diff
@@ -1,10 +1,210 @@
...
+
+**Operating systems**
+
+- Red Hat Enterprise Linux 8, 64-bit
+
+**Software**
+
+- [CMake 3.13 or higher, 64-bit](https://cmake.org/download/)
+- GCC 8.2.0
+- [Python 3.7 - 3.10, 64-bit](https://www.python.org/downloads/windows/)
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-yum.md:21]: Corrected.

File: docs/install_guides/installing-openvino-apt.md [hunk 11]
```diff
@@ -1,10 +1,228 @@
...
+**Operating Systems**
+
+- Ubuntu 18.04 long-term support (LTS), 64-bit
+- Ubuntu 20.04 long-term support (LTS), 64-bit
+
+**Software**
+
+- [CMake 3.13 or higher, 64-bit](https://cmake.org/download/)
+- GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
+- [Python 3.7 - 3.10, 64-bit](https://www.python.org/downloads/windows/)
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md:22]: Corrected.

File: docs/install_guides/installing-openvino-yum.md (hunk 4 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-yum.md:64]: Shall we specify here, which system locations they are?

File: docs/install_guides/installing-openvino-yum.md (hunk 2 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-yum.md:192]: Have removed, commit underway.

File: docs/install_guides/installing-model-dev-tools.md (hunk 3 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-model-dev-tools.md:100]: > BTW, dev tools usually stand for Developer Tools, not Development. Hope, we can have the same notation.

I have to argue that it is Development Tools in the Intel name database. :)
image

Comment (sgolebiewski-intel) [reply docs/install_guides/installing-model-dev-tools.md:106]: Revision applied with https://github.com/openvinotoolkit/openvino/pull/14492/commits/45f952fd3ddd82d0eaa305e1a09e5511169b7d44

File: docs/install_guides/installing-openvino-yum.md (hunk 1 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-yum.md:138]: Removed with https://github.com/openvinotoolkit/openvino/pull/14492/commits/bce5a845f8329dbc9b09da05817f04efb5acf19d

File: docs/install_guides/installing-openvino-apt.md [hunk 12]
```diff
@@ -1,10 +1,228 @@
...
+   For example:
+
+   .. code-block:: sh
+
+      sudo apt autoremove openvino-2022.3.0
+
+@endsphinxdirective
+
+
+### Step 3 (Optional): Install OpenCV from APT
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md]: Removed OpenCV section with https://github.com/openvinotoolkit/openvino/pull/14492/commits/bce5a845f8329dbc9b09da05817f04efb5acf19d

File: docs/install_guides/installing-openvino-apt.md [hunk 13]
```diff
@@ -1,10 +1,228 @@
...
+   <!--For more details, see the `Get Started page for Intel® Neural Compute Stick 2 <https://software.intel.com/en-us/neural-compute-stick/get-started>`.-->
+
+.. tab:: VPU
+
+   To install and configure your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, see the :ref:`VPU Configuration Guide <vpu guide>`.
+   After configuration is done, you are ready to run the verification scripts with the HDDL Plugin for your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs.
+
+   .. warning::
+      While working with either HDDL or NCS, choose one of them as they cannot run simultaneously on the same machine.
+
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md]: Removed with https://github.com/openvinotoolkit/openvino/pull/14492/commits/d6d6ba4e398f70b5b23adbab8dac9da8c6dc142f

File: docs/install_guides/installing-model-dev-tools.md (hunk 5 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-model-dev-tools.md:128]: Have corrected with the following commit: https://github.com/openvinotoolkit/openvino/pull/14492/commits/bf2e25af8fcf9faa12e6488ecfa5771de6cebe9d

File: docs/install_guides/installing-openvino-apt.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md]: Have updated the disclaimer: removed the sentence about <INSTALL_DIR>/licensing with the following commit: https://github.com/openvinotoolkit/openvino/pull/14492/commits/d3c97e6c1a3a06585b4ed92a61052df88d6e50be

File: docs/install_guides/installing-openvino-apt.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md:177]: Added build samples section with https://github.com/openvinotoolkit/openvino/pull/14492/commits/80349e5bbdca89fe0c636133eeb9b157f6525165

File: docs/install_guides/installing-openvino-apt.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md]: > @sgolebiewski-intel We are missing two pages to be updated from the list I sent you: the installation overview page and the openvino_docs_get_started_get_started_demos page.

Have edited those two files accordingly.

File: docs/install_guides/installing-openvino-apt.md [hunk 14]
```diff
@@ -1,10 +1,228 @@
...
\ No newline at end of file
+> **IMPORTANT**: By downloading and using this container and the included software, you agree to the terms and conditions of the [software license agreements](https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf). Please review the content inside the `<INSTALL_DIR>/licensing` folder for more details.
+
+## System Requirements
+
+The complete list of supported hardware is available in the [Release Notes](https://software.intel.com/content/www/us/en/develop/articles/openvino-relnotes.html).
+
+**Operating Systems**
+
+- Ubuntu 18.04 long-term support (LTS), 64-bit
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md]: Added info about the architecture: https://github.com/openvinotoolkit/openvino/pull/14492/commits/f06f3d116f0e927678c3935b78cc3d9e8f1ab78b

File: docs/install_guides/installing-openvino-yum.md (hunk 4 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-yum.md]: We will specify the proper directories in this article, once we have had substantial information. For now, I have removed these two lines with https://github.com/openvinotoolkit/openvino/pull/14492/commits/5fbef2ce42dd812b17f51e622fa15c1284a140cf

File: docs/install_guides/installing-openvino-apt.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md]: We will specify the proper directories in this article, once we have had substantial information. For now, I have removed these two lines with https://github.com/openvinotoolkit/openvino/pull/14492/commits/bd12b0352ae84e57800e14b0b6569bc320c32589

File: docs/install_guides/installing-openvino-yum.md [hunk 15]
```diff
@@ -1,10 +1,210 @@
...
-* [Install OpenVINO from PyPI](installing-openvino-pip.md)
-* [Install OpenVINO with Docker](installing-openvino-docker-linux.md)
-* [Build From Source](https://github.com/openvinotoolkit/openvino/wiki/BuildingCode)
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. If you want to develop or optimize your models with OpenVINO, see [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
 
-The other installation methods are temporarily unavailable.
\ No newline at end of file
+> **IMPORTANT**: By downloading and using this container and the included software, you agree to the terms and conditions of the [software license agreements](https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf). Please review the content inside the `<INSTALL_DIR>/licensing` folder for more details.
+
+## System Requirements
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-yum.md:9]: Could you be more specific?

File: docs/install_guides/installing-openvino-apt.md [hunk 16]
```diff
@@ -1,10 +1,216 @@
...
+   apt-cache search openvino
+   ```
+
+
+### Step 2: Install OpenVINO Runtime Using the APT Package Manager
+
+#### Install OpenVINO Runtime
+
+@sphinxdirective
+
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-apt.md:98]: Let's also note, that users can use `–no-install-recommends` option to install only required packages, but in this case samples cannot be compiled because build tools are not installed.

File: docs/install_guides/installing-openvino-apt.md [hunk 17]
```diff
@@ -1,10 +1,216 @@
...
+  Optimized for these processors:
+
+  * 6th to 12th generation Intel® Core™ processors and Intel® Xeon® processors
+  * 3rd generation Intel® Xeon® Scalable processor (formerly code named Cooper Lake)
+  * Intel® Xeon® Scalable processor (formerly Skylake and Cascade Lake)
+  * Intel Atom® processor with support for Intel® Streaming SIMD Extensions 4.1 (Intel® SSE4.1)
+  * Intel Pentium® processor N4200/5, N3350/5, or N3450/5 with Intel® HD Graphics
+  * Intel® Iris® Xe MAX Graphics
+  * Intel® Neural Compute Stick 2
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md]: Removed with https://github.com/openvinotoolkit/openvino/pull/14492/commits/96f1065b2445b7cea71e2c21b8788befda669199

File: docs/install_guides/installing-openvino-yum.md [hunk 18]
```diff
@@ -1,10 +1,199 @@
...
+
+  Optimized for these processors:
+
+  * 6th to 12th generation Intel® Core™ processors and Intel® Xeon® processors
+  * 3rd generation Intel® Xeon® Scalable processor (formerly code named Cooper Lake)
+  * Intel® Xeon® Scalable processor (formerly Skylake and Cascade Lake)
+  * Intel Atom® processor with support for Intel® Streaming SIMD Extensions 4.1 (Intel® SSE4.1)
+  * Intel Pentium® processor N4200/5, N3350/5, or N3450/5 with Intel® HD Graphics
+  * Intel® Iris® Xe MAX Graphics
+  * Intel® Neural Compute Stick 2
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-yum.md]: Removed with https://github.com/openvinotoolkit/openvino/pull/14492/commits/96f1065b2445b7cea71e2c21b8788befda669199

File: docs/install_guides/installing-openvino-yum.md [hunk 19]
```diff
@@ -1,10 +1,199 @@
...
+   .. code-block:: sh
+
+      sudo yum autoremove openvino-2022.3.0
+
+@endsphinxdirective
+
+
+### Step 3 (Optional): Install Software Dependencies
+
+After you have installed OpenVINO Runtime, if you decided to [install OpenVINO Model Development Tools](installing-model-dev-tools.md), make sure that you install external software dependencies first.
```
[the remark this answers is not in the document]
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-yum.md:155]: Have added the section to the other document with https://github.com/openvinotoolkit/openvino/pull/14492/commits/8e9ddf0a49062d406e553d41f26e1f818e509811

File: docs/install_guides/installing-openvino-apt.md (hunk 16 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-apt.md:97]: Have applied a note regarding this option with https://github.com/openvinotoolkit/openvino/pull/14492/commits/fd16a27d39aff668654e462b42cb8601909aad9b

**Review by ilya-lavrenov — APPROVED** [review]
approved, but better to move not with `--no-install-recommends` not the end of step 2. install OpenVINO

**Comment by sgolebiewski-intel** [comment]
> approved, but better to move not with `--no-install-recommends` not the end of step 2. install OpenVINO

Have moved the note with https://github.com/openvinotoolkit/openvino/pull/14492/commits/c6c147d2c9308c5354e49754a95fbff2f3390cab
